core/src/main/java/com/datastrato/gravitino/Entity.java

```diff
@@ -43,6 +45,37 @@ public static EntityType fromShortName(String shortName) {
       }
       throw new IllegalArgumentException("Unknown entity type: " + shortName);
     }
+
+    /**
+     * Returns the parent entity types of the given entity type. The parent entity types are the
+     * entity types that are higher in the hierarchy than the given entity type. For example, the
+     * parent entity types of a table are metalake, catalog, and schema. (Sequence: root to leaf)
+     *
+     * @param entityType The entity type for which to get the parent entity types.
+     * @return The parent entity types of the given entity type.
+     */
+    public static List<EntityType> getParentEntityTypes(EntityType entityType) {
+      switch (entityType) {
+        case METALAKE:
+          return ImmutableList.of();
+        case CATALOG:
+        case USER:
+        case GROUP:
```

Comment:
@qqqttt123 , Can you confirm if these changes are acceptable?

Reply:
Now, My nameIdentifier is `metalake."system"."user".user1` and `metalake."system"."group".group1`.